tieredstore/src/main/java/org/apache/rocketmq/tieredstore/TieredMessageStore.java

```diff
@@ -99,6 +99,11 @@ public boolean viaTieredStorage(String topic, int queueId, long offset) {
 
     public boolean viaTieredStorage(String topic, int queueId, long offset, int batchSize) {
         TieredMessageStoreConfig.TieredStorageLevel deepStorageLevel = storeConfig.getTieredStorageLevel();
+
+        if (deepStorageLevel.check(TieredMessageStoreConfig.TieredStorageLevel.FORCE)) {
```

Comment:
> We should add a comment for the `FORCE` level. It will force `viaTieredStorage` retuning true, but it can not prevent falling back to the default store when failing to fetch messages from tiered storage.

Yep. I have added some comments for `TieredStorageLevel`